### ISSUE TITLE
Refresh rate in full screen mode is restricted to 17 Hz

### ISSUE THREAD

**Issue by NanoSector** [utmapp/UTM#3864]
**Describe the issue**
The refresh rate in full screen mode with the resolution automatically set by UTM is restricted to 17 Hz causing extremely choppy animations. Changing the resolution to something else in the guest OS unlocks higher refresh rates.

This has been tested in Fedora 35 Workstation and Fedora 36 Beta Workstation, both exhibit the issue, both with X11 and Wayland, with and without GPU acceleration enabled.

The host device is a 14" MacBook Pro without external displays.

The attached debug log and config are from Fedora 35 Workstation booting into a GNOME Wayland session.

**Configuration**
* UTM Version: 3.1.5
* OS Version: 12.3
* Intel or Apple Silicon? Apple M1 Pro, 10 CPU & 16 GPU

**Debug log**
[debug.log](https://github.com/utmapp/UTM/files/8404706/debug.log)

**Upload VM**
[config.plist.zip](https://github.com/utmapp/UTM/files/8404704/config.plist.zip)

**Comment by Terabyte1385** [comment]
Experiencing the same issues here, MacBook Air (Apple M1 with 16GB RAM). Guest OS is Windows 11 Pro for ARM, and UTM version is 3.1.5. A difference here is that, even with Retina mode off, UTM still struggles with even simple animations such as the loading bar on the Welcome screen and the new window animations. Cursor movements sometimes feel very slow. However, videos from YouTube play smoothly (Edge is the browser used if it matters).

**Comment by astroboylrx** [comment]
> Experiencing the same issues here, MacBook Air (Apple M1 with 16GB RAM). Guest OS is Windows 11 Pro for ARM, and UTM version is 3.1.5. 

Not sure if this is the same issue, but my Windows 11 Pro for ARM stuck at 1024x768 for the "active signal mode." So even if I choose 4k desktop mode, the text looks blurred. ("List All Modes" in display adapter properties only changes the desktop mode and has no effect on the active signal mode.)
![image](https://user-images.githubusercontent.com/3999966/168188039-df2b9a37-8d6c-41f1-a0df-8d5a55ab07af.png)

Any suggestions would be greatly appreciated!

**Comment by sainnhe** [comment]
> > Experiencing the same issues here, MacBook Air (Apple M1 with 16GB RAM). Guest OS is Windows 11 Pro for ARM, and UTM version is 3.1.5.
> 
> Not sure if this is the same issue, but my Windows 11 Pro for ARM stuck at 1024x768 for the "active signal mode." So even if I choose 4k desktop mode, the text looks blurred. ("List All Modes" in display adapter properties only changes the desktop mode and has no effect on the active signal mode.) ![image](https://user-images.githubusercontent.com/3999966/168188039-df2b9a37-8d6c-41f1-a0df-8d5a55ab07af.png)
> 
> Any suggestions would be greatly appreciated!

Open UTM App, select Windows in left bar, click settings icon in top right corner, select display, there is an option to enable hidpi mode in bottom.

**Comment by Belluxx** [comment]
@astroboylrx I am facing the same issue, did you find a solution? 
I also opened an issue about it #4607

**Comment by astroboylrx** [comment]
> @astroboylrx I am facing the same issue, did you find a solution? I also opened an issue about it #4607

Not really. I went to use VMware Fusion Tech Preview. Resolution-wise, it works great.

**Comment by Terabyte1385** [comment]
Any updates on this? It's driving me crazy!

**Comment by Belluxx** [comment]
> Try using the iso from uupdump instead of from microsoft.com .

I tried both and experienced the same issue
